load.php

```diff
@@ -76,7 +87,7 @@ function( $module_settings ) {
  * @return array Associative array of module settings keyed by module slug.
  */
 function perflab_get_module_settings() {
-	return (array) get_option( PERFLAB_MODULES_SETTING, array() );
+	return (array) get_option( PERFLAB_MODULES_SETTING );
```

Comment:
When this one is called for the first time this would return 1 element array:

```php
array( false )
```

Which is handled in other parts of the code by using `array_filter` but I would suggest to make sure this returns consistent results instead.

```suggestion
	return array_filter( (array) get_option( PERFLAB_MODULES_SETTING ) );
```

So when this is called when the setting does not exists this would return an empty array instead.

Reply:
@mitogh It wouldn't return `array( false )`, since this setting has a registered default now (see above). The point of removing the second parameter here was to ensure the registered default is used. So the `get_option` call here can actually never result in `false` based on that.

Reply:
Good point :+1: